src/Storage/DatabaseStorage.php

```diff
@@ -179,7 +179,9 @@ protected function upsertCount(array $values): int
             [
                 'value' => match ($driver = $this->connection()->getDriverName()) {
                     'mariadb', 'mysql' => new Expression('`value` + values(`value`)'),
-                    'pgsql', 'sqlite' => new Expression('"pulse_aggregates"."value" + "excluded"."value"'),
+                    'pgsql', 'sqlite' => new Expression(<<<SQL
+                        {$this->wrap('pulse_aggregates.value')} + "excluded"."value"
```

Comment:
I opted to leave the `"excluded"."value"` "raw" because `excluded` is a special keyword in this scenario and should not be prefixed.